contrib/setup_testbed.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
```

Comment:
Autoformat this file with black

Reply:
And use setup-testbed.py instead of setup_testbed.py. We use the minus sign almost everywhere in the file names.

Reply:
Reformatted and renamed :-)

Reply:
Added "black" to the requirements and reformatted.